### ISSUE TITLE
Unable to set CenterX and CenterY to 0

### ISSUE THREAD

**Issue by chamitha** [Minitour/EasyNotificationBadge#15]
It is not possible to set the centerY (and centerX) offsets to 0. The following code prevents this. However 0 is a valid value to center align precisely. 

```
 //The distance from the center of the view (vertically)
let centerY = appearance.distanceFromCenterY == 0 ? -(bounds.size.height / 2) : appearance.distanceFromCenterY

 //The distance from the center of the view (horizontally)
let centerX = appearance.distanceFromCenterX == 0 ? (bounds.size.width / 2) : appearance.distanceFromCenterX
```

A workaround is to set `distanceFromCenterY` or `distanceFromCenterX` to a very small non zero value such as 0.01 but it feels yucky!

**Comment by Minitour** [comment]
Yeah that’s right. This was written 4 years ago with not much thought into it. It would be possible to make the variables optional, then simply not apply them if they are nil.

**Comment by Minitour** [comment]
Fixed in version `1.2.4`
